src/app/state/staking/saga.ts

```diff
@@ -84,8 +84,15 @@ function* loadDebondingDelegations(publicKey: Uint8Array) {
 }
 
 function* refreshValidators() {
+  // Fetch active validator set size.
+  const nic = yield* call(getOasisNic)
+  const consensusParams = yield* call([nic, nic.schedulerConsensusParameters], consensus.HEIGHT_LATEST)
+
+  // Fetch validators.
   const { accounts } = yield* call(getExplorerAPIs)
-  const validators = yield* call([accounts, accounts.getValidatorsList], {})
+  const validators = yield* call([accounts, accounts.getValidatorsList], {
+    limit: consensusParams.max_validators,
+  })
```

Comment:
Does max_validators include inactive validators?

Otherwise this isn't exactly correct: When fetching 500, status=inactive validators were at indexes `[86, 98, 113, 114, 118, 119, 120, 121, 123, 124, 125, 126, 127, 130, 131, 132, 133]`, so this loses active validators at 108 and 109

Reply:
`max_validators` contains current maximum allowed size of active validator set. If a validator is not active, there is some expiration period before it gets evicted from the active validator set, so it may be a valid assumption the validator is still in the active set although inactive currently. The same goes for active validators - they may be registered as validators, but may not be in the active validator set yet.

So I was unable to find out how oasis-explorer generates the list of validators here https://github.com/everstake/oasis-explorer/blob/master/dao/clickhouse/validators.go#L13-L17 and how it updates it here https://github.com/everstake/oasis-explorer/blob/master/dao/clickhouse/validators.go#L207 and combining it with metadata registry of entities here https://github.com/oasisprotocol/metadata-registry. I suggest we open a separate issue regarding those borderline cases. For now I would simply return the top-k validators where k=max_validators parameter and hide the rest.

Reply:
I discussed this with @kostko and the correct approach would be first to call `schedulerGetValidators(height)` which gives you the actual validator set (no inactive validators!) and then intersect that list with the one from oasis-explorer to obtain detailed information (icon, name, escrow, fee etc.).

Reply:
Nope, there are 116 (and block explorer API doesn't support filtering by active status)

Reply:
> I discussed this with @kostko and the correct approach would be first to call schedulerGetValidators(height) which gives you the actual validator set (no inactive validators!)

I would prefer to list all validators, i.e. all entities that have a validator node registered, even if they are not in the current validator set.

But it would be nice to somehow visually distinguish those that are not active versus active ones.

Reply:
116 validators with status=active are marked as green, vs 18 red.
![image](https://user-images.githubusercontent.com/3758846/142295895-a453bf3c-7771-4a34-ad4b-f6930d8451ed.png)

If I understand, you are proposing another symbol for bottom 6 status=active validators?

Reply:
For now, I'll merge this by fetching 500 (with Matevž's permission)